vm/proxyapp/init.go

```diff
@@ -49,8 +51,17 @@ type subProcessCmd interface {
 	Wait() error
 }
 
+// Config is valid if at least cmd or rpc_server_uri specified.
 type Config struct {
-	Command        string          `json:"cmd"`
+	// cmd is the optional command needed to initialize plugin.
+	// By default we'll connect to its std[in, out, err].
+	Command string `json:"cmd"`
+	// rpc_server_uri is used to specify plugin endpoint address.
+	// if not specified, we'll connect to the plugin by std[in, out, err].
+	RPCServerURI string `json:"rpc_server_uri"`
+	// logs_uri is an optional parameter
+	LogsURI string `json:"logs_uri"`
```

Comment:
Do we really need this? I feel it will complicate the logic. We will just proxy the logs, but since we don't necessary control the service lifetime (it runs when the manager doesn't) it will need to log somewhere else anyway.
An additional long poll rpc request to add logs to manager log may be simpler.

Reply:
And I'd love to go your way.
The limitation I'm worrying about is the limited ability of the RPC server to send crash dumps.
My proposal is to wrap remote server application by netcat ([example](https://superuser.com/questions/1533731/use-netcat-to-listen-on-a-port-and-send-output-from-a-command-when-a-client-conn)) and potentially get full logs.

Both ways looks good for me.

Reply:
Having two options let's go easier way now and add complexity when needed.
Let's go logging RPC way.